### DISCUSSION TITLE
Изменения

### DISCUSSION THREAD

**Discussion by yuipp** [shvchk/padavan-builder-workflow#47] [Вопросы и ответы] [answered]
Привет, свежие изменения смотреть на gitlab?

**Comment by shvchk** [comment, marked as answer]
Изменения в прошивке — да, по умолчанию используется [репо прошивки на гитлабе](https://gitlab.com/hadzhioglu/padavan-ng), см. `PADAVAN_REPO` в файле `variables`. Изменения в сборщике — здесь, в этом репозитории, но они довольно редки.
